econml/ortho_forest.py

```diff
@@ -330,7 +374,26 @@ def _get_weights(self, X_single):
                 w2[unique] += leaf_weight * counts
             else:
                 w2[weight_indexes] += leaf_weight
-        return (w1 / self.n_trees, w2 / self.n_trees)
+        return (w1 / len(tree_range), w2 / len(tree_range))
+
+    def _get_blb_indices(self, X):
+        """Get  data indices for every tree under the little bags split."""
+        # Define subsample size
+        subsample_size = X.shape[0] // 2
+        if not self.bootstrap:
+            if self.subsample_ratio > 1.0:
+                # Safety check
+                self.subsample_ratio = 1.0
+            subsample_size = int(self.subsample_ratio * subsample_size)
+        subsample_ind = []
+        # Draw points to create little bags
+        for it in range(self.n_slices):
+            half_sample_inds = self.random_state.choice(
+                X.shape[0], X.shape[0] // 2, replace=False)
```

Comment:
```suggestion
                X.shape[0], subsample_size, replace=False)
```

Reply:
This change is not quite right because `subsample_size` is smaller than `X.shape[0] // 2` if subsampling is enabled (which is the default). I did make a modification for the `replace=False` argument in line 395 since it would not have done the right thing for `bootstrap=True`.